packages/harmony/src/foundations/color/semantic.ts

```diff
@@ -0,0 +1,104 @@
+
+export type SemanticColors = {
+  textIcon: {
+    heading: GradientColor
+    default: string
+    subdued: string
+    disabled: string
+  }
+  background: {
+    default: string
+    white: string
+    surface1: string
+    surface2: string
+  }
+  border: {
+    default: string
+    strong: string
+  }
+  focus: string
+  status: {
+    error: string
+    warning: string
+    success: string
+  }
+}
+
+export const semanticTheme: Record<Theme, SemanticColors> = {
+  day: {
+    textIcon: {
+      heading: primitiveTheme.day.special.gradient,
+      default: primitiveTheme.day.neutral.neutral,
+      subdued: primitiveTheme.day.neutral.n400,
+      disabled: primitiveTheme.day.neutral.n150
+    },
+    background: {
+      default: primitiveTheme.day.special.background,
+      white: primitiveTheme.day.special.white,
+      surface1: primitiveTheme.day.neutral.n25,
+      surface2: primitiveTheme.day.neutral.n100
+    },
+    border: {
+      default: primitiveTheme.day.neutral.n100,
+      strong: primitiveTheme.day.neutral.n150
+    },
+    focus: primitiveTheme.day.secondary.secondary,
+    status: {
+      error: primitiveTheme.day.special.red,
+      warning: primitiveTheme.day.special.orange,
+      success: primitiveTheme.day.special.green
+    }
+  },
+  dark: {
+    textIcon: {
+      heading: primitiveTheme.dark.special.gradient,
+      default: primitiveTheme.dark.neutral.neutral,
+      subdued: primitiveTheme.dark.neutral.n400,
+      disabled: primitiveTheme.dark.neutral.n150
+    },
+    background: {
+      default: primitiveTheme.dark.special.background,
+      white: primitiveTheme.dark.special.white,
+      surface1: primitiveTheme.dark.neutral.n25,
+      surface2: primitiveTheme.dark.neutral.n100
+    },
+    border: {
+      default: primitiveTheme.dark.neutral.n100,
+      strong: primitiveTheme.dark.neutral.n150
+    },
+    focus: primitiveTheme.dark.secondary.secondary,
+    status: {
+      error: primitiveTheme.dark.special.red,
+      warning: primitiveTheme.dark.special.orange,
+      success: primitiveTheme.dark.special.green
+    }
+  },
+  matrix: {
+    textIcon: {
+      heading: primitiveTheme.matrix.special.gradient,
+      default: primitiveTheme.matrix.neutral.neutral,
+      subdued: primitiveTheme.matrix.neutral.n400,
+      disabled: primitiveTheme.matrix.neutral.n150
+    },
+    background: {
+      default: primitiveTheme.matrix.special.background,
+      white: primitiveTheme.matrix.special.white,
+      surface1: primitiveTheme.matrix.neutral.n25,
+      surface2: primitiveTheme.matrix.neutral.n100
+    },
+    border: {
+      default: primitiveTheme.matrix.neutral.n100,
+      strong: primitiveTheme.matrix.neutral.n150
+    },
+    focus: primitiveTheme.matrix.secondary.secondary,
+    status: {
+      error: primitiveTheme.matrix.special.red,
+      warning: primitiveTheme.matrix.special.orange,
+      success: primitiveTheme.matrix.special.green
+    }
+  }
+}
```

Comment:
We should be able to use getter functions or something here to leverage the fact that the same primitive colors will be used for each of the semantic colors between themes. Down to discuss what would work best here

Reply:
yeah def agree! i was thinking it might be good to be extra wet here in case there were random changes for a theme, and to be extra explicit?

Reply:
I asked design specifically about this bc i was worried about the same thing and Sammie said that they should always be the same, but I'm down to be explicit here just in case.

Reply:
word yeah either way imo

Reply:
it's way cleaner how you said, updating...